regolith/tools.py

```diff
@@ -893,3 +893,51 @@ def group_member_ids(ppl_coll, grpname):
                 if position.get("group", None) == grpname:
                     grpmembers.add(person["_id"])
     return grpmembers
+
+def fragment_retrieval(coll, fields, fragment, case_sensitive = True):
```

Comment:
I think default is true for fuzzy retrieval, but here I would rather have False as the default